syncmaster/schemas/v1/groups.py

```diff
@@ -2,8 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 from pydantic import BaseModel
 
-from app.api.v1.schemas import NameConstr, PageSchema
-from app.db.models import GroupMemberRole
+from syncmaster.db import GroupMemberRole
+from syncmaster.schemas.v1.schemas import NameConstr, PageSchema
```

Comment:
```suggestion
from syncmaster.schemas.v1 import NameConstr, PageSchema
```

Давай так. Ну или внутри v1 должно быть деление на схемы, связанные с какой-то определенной сущностью, например `v1.page`, `v1.group` и т.п.